### DISCUSSION TITLE
Production app database

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#2995] [Help] [answered]
Hello All,

Have a few questions:

- Could you please advise on the way to access the app database in a published app?

- What are the options available for data export and backup?

- Is there a way to access the Budibase database from an external app?

- The internal database is CouchDB correct?

Catching up after a long time! :)

Thank you!

**Comment by mjashanks** [comment, marked as answer]
> Could you please advise on the way to access the app database in a published app?

So, assuming that you are self-hosting..

Budibase runs on CouchDB. You can access here: `http://your-domain-and-port/db/_utils/#`. Your username and password will be contained within your `.env` file, which lives in the same directory as your `docker-compose.yaml`.

> What are the options available for data export and backup?

Right now, its manual - you'll have to backup CouchDb yourself. We hope to offer this soon, but not for a few months.

>  Is there a way to access the Budibase database from an external app?

Budibase has it's own API, but it's not well documented - so not officially supported at the moment. Again, in time we will support this properly. For now, you could watch the network requests that get sent from the client app, to do some basic things.

> Where did the login screen for a published app go?

It's now one login for the whole deployment, not per app. You can read about this here: https://github.com/Budibase/budibase/discussions/1975
